test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -421,4 +571,72 @@ public static void forceFail(String reason) {
         testFailedReason = FAILURE_REASON + reason;
         latch.countDown();
     }
+
+    public static class Builder {
+        private String title;
+        private String instructions;
+        private long testTimeOut;
+        private int rows;
+        private int columns;
+        private boolean enableScreenCapture = false;
```

Comment:
@lawrence-andrew  you missed updating var `enableScreenCapture` to `screenCapture` here. You can either update only the method name  to screenCapture() and retain same booolean var name - enableScreenCapture or update two other places where they are being used.

Reply:
Its better to keep method name and var name separately.

Reply:
They had the same name; Java allows the same name for a field and a method. For consistency with other methods and fields, I agree with @honkar-jdk and I'm for renaming it.